pontoon/base/static/js/fluent_interface.js

```diff
@@ -613,27 +696,37 @@ var Pontoon = (function (my) {
           var source = object.original || object.string;
 
           if (!source) {
-            return response;
+            return fallback;
           }
 
           var ast = fluentParser.parseEntry(source);
+          var tree;
 
+          // Value: use entire ast
           if (ast.value) {
-            response = this.serializePlaceables(ast.value.elements);
+            tree = ast;
           }
-          // Attributes
-          else {
-            var attributes = ast.attributes;
-            if (attributes && attributes.length) {
-              response = this.serializePlaceables(attributes[0].value.elements);
-            }
+
+          // Attributes: use ast of the first attribute
+          else if (ast.attributes.length) {
+            tree = ast.attributes[0];
           }
-          // Plurals
-          if (this.isPlural(ast)) {
-            var variants = ast.value.elements[0].variants;
-            response = this.serializePlaceables(variants[0].value.elements);
+
+          // Simple string: use entire value
+          var elements = tree.value.elements;
+
+          // String with variants: use default variant
+          try {
+            elements = elements[0].variants.filter(function (item) {
```

Comment:
Still concerned about the assumption that it's the top-level element that makes this string non-simple.